photutils/psf/photometry.py

```diff
@@ -230,8 +230,11 @@ def do_photometry(self, image, init_guesses=None):
         output_tab : `~astropy.table.Table` or None
             Table with the photometry results, i.e., centroids and
             fluxes estimations and the initial estimates used to start
-            the fitting process.
-            None is returned if no sources are found in ``image``.
+            the fitting process. Uncertainties on the fitted parameters are
+            reported provided that the fitter object contains a dictionary
+            called ``fit_info`` with ``param_cov`` as key to access the
+            covariance matrix. If this dictionary is not available,
+            uncertainties columns will be filled with ``np.nan``.
```

Comment:
Is there a reason for filling with `nan` instead of just not having the column?  The latter is easier to check for...